test/e2e/framework.go

```diff
@@ -104,6 +104,7 @@ const (
 	nameSuffixLength int = 8
 
 	agnhostImage        = "projects.registry.vmware.com/antrea/agnhost:2.26"
+	agnhostImage229     = "k8s.gcr.io/e2e-test-images/agnhost:2.29"
```

Comment:
can we can we just update `agnhostImage`, do we need 2 different versions for the tests?